groth16/src/lib.rs

```diff
@@ -1,5 +1,285 @@
+pub mod export_solidity_verifier;
 pub mod groth16;
 
 pub use bellman_ce::pairing::ff;
 pub use ff::*;
 pub use franklin_crypto::bellman as bellman_ce;
+
+mod parse {
```

Comment:
mod is not necessary here, instead, you can move those converting functions into a single file, like json_export.rs